neo/Consensus/ConsensusContext.cs

```diff
@@ -95,7 +95,7 @@ public void Deserialize(BinaryReader reader)
             Reset(0);
             if (reader.ReadUInt32() != Block.Version) throw new FormatException();
             if (reader.ReadUInt32() != Block.Index) throw new InvalidOperationException();
-            Block.Timestamp = reader.ReadUInt32();
+            Block.Timestamp = reader.ReadInt64();
```

Comment:
ulong?

Reply:
Can be, @lock9 already suggested. I thought it would not be necessary, but now that you and @igormcoelho also mentioning...aehuaheuaea